### PR TITLE
Roadmap

### PR DESCRIPTION
**Pull request by viv9k**
### Functionality:
Raodmap phase 1 development

### Solution:
Added roadmap table and milestones can now be seen as in roadmap table.
General code improvement

## WtId:
<-- please enter your WorktezId -->

### Risk level:
- [x] high 
- [ ] medium
- [ ] low

### How to test:
site

### CODE REVIEW

**Review by simran142002 — COMMENTED** [review]
We can remove date service completely right?

File: src/app/header/header.component.ts [hunk 1]
```diff
@@ -32,8 +34,40 @@ export class HeaderComponent implements OnInit {
...
+      {name: 'Market activation', events:  [
+                                    {name: 'Market activity', startDate: new Date('2021-02-15'),  endDate: new Date('2021-02-28')} as RoadmapEvent
+                                    ],
+        mileStones: [{name:'Funding round complete', date: new Date('2021-01-28')} as RoadmapMileStone]
+      } as RoadmapRow,
+      {name: 'Google ads campaign', events:  [
+                                    {name: 'Busy period', startDate: new Date('2021-03-02'),  endDate: new Date('2021-03-15')} as RoadmapEvent
+                              ]      
+      } as RoadmapRow,
+      {name: 'Client feedback', events:  [
```
Comment (sanjaykrishna1203) [inline src/app/header/header.component.ts:60]: I can't find where this data is being used...

File: src/app/body/roadmap/roadmap.component.ts [hunk 2]
```diff
@@ -0,0 +1,77 @@
+/***********************************************************
+ * Copyright (C) 2022
+ * Worktez
+ * Author : Simran Nigam <nigamsimran14@gmail.com>
+ *
```
Comment (sanjaykrishna1203) [inline src/app/body/roadmap/roadmap.component.ts:5]: Author name should be changed

File: src/app/body/roadmap/roadmap.component.html [hunk 3]
```diff
@@ -0,0 +1,20 @@
+<!-- /***********************************************************
+ * Copyright (C) 2022
+ * Worktez
+ * Author : Simran Nigam <nigamsimran14@gmail.com>
+ *
```
Comment (sanjaykrishna1203) [inline src/app/body/roadmap/roadmap.component.html:5]: Same here

**Pull request merged**